media/js/devreg/l10n.js

```diff
@@ -225,11 +225,12 @@ $(document).ready(function () {
         $("#change-locale").text(current.text());
         $(".trans").each(function () {
             var $el = $(this),
+                $ni = null,
```

Comment:
This is style more than a jshint thing but these should all be vars on separate lines.